config/locales/en.yml

```diff
@@ -376,7 +376,7 @@ en:
         course_start_date_hint: "You can also select this option if you have already started"
         maths_understanding_of_approach_hint: "For example, a reference from a professional who has supported your mastery professional development."
         lead_provider_id: These are the training providers who provide <b>%{course_name}</b>. Providers may have different entry requirements.
-        trn: Your TRN is between 6 and 7 characters long. It may begin with 'RP'
+        trn: Your TRN is between 6 and 7 characters long.
```

Comment:
TRNs [should be 7 digits](https://www.gov.uk/guidance/teacher-reference-number-trn):

> A TRN is a 7-digit number that uniquely identifies you in the education sector in England

Reply:
Updated the validations